AppCheckCore/Tests/Unit/Swift/AppCheckAPITests.swift

```diff
@@ -29,23 +29,21 @@ final class AppCheckAPITests {
 
     // MARK: - AppAttestProvider
 
-    #if os(iOS) || os(tvOS) || os(watchOS)
-      if #available(iOS 14.0, tvOS 15.0, watchOS 9.0, *) {
-        // TODO(andrewheard): Add `requestHooks` in API tests.
-        if let provider = AppCheckCoreAppAttestProvider(
-          serviceName: serviceName,
-          resourceName: resourceName,
-          baseURL: nil,
-          apiKey: apiKey,
-          keychainAccessGroup: nil,
-          requestHooks: nil
-        ) {
-          provider.getToken { token, error in
-            // ...
-          }
+    if #available(iOS 14.0, macOS 11.3, macCatalyst 14.5, tvOS 15.0, watchOS 9.0, *) {
```

Comment:
Is this intentionally adding macOS?

Reply:
Yes, I think I overcautiously omitted macOS when splitting this off from FirebaseAppCheck. In the original code it was [`#if TARGET_OS_IOS`](https://github.com/firebase/firebase-ios-sdk/blob/2258c07ecb29d1d936c720d57a15b066df962de8/FirebaseAppCheck/Tests/Unit/Swift/AppCheckAPITests.swift#L28) (always false in Swift). I changed it to `#if os(iOS) || os(tvOS) || os(watchOS)` when I forked it but the API is also available on macOS (including Catalyst) so I just dropped the `os()` conditionals entirely. This also allows it to work on visionOS since it falls through the `*` in the `if #available`.